src/components/search/searchHeader/SearchHeader.tsx

```
@@ -1,6 +1,6 @@
 import Icon from '@assets/svgs';
 import SearchBar from '@components/search/searchBar/SearchBar';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)